README.md

```diff
@@ -82,11 +82,11 @@ require 'vendor/autoload.php';
 // If you are not using Composer
 // require("path/to/sendgrid-php/sendgrid-php.php");
 
-$from = new SendGrid\Email("Example User", "test@example.com");
+$from = new \Helpers\Email("Example User", "test@example.com");
```

Comment:
I'm not sure this is the right way to name these since we plan to create more helpers (for example `ContactDB`. I think this helper should be `Mail`. In that case, we should rename the existing `Mail` function to `SendGridMessage` since it's job is to create a SendGridMessage object that will then be used to send out the email. Check out the [naming conventions here](https://github.com/sendgrid/sendgrid-csharp/tree/master/src/SendGrid/Helpers/Mail/Model), I think these are good.